pandas/core/generic.py

```diff
@@ -10512,6 +10512,7 @@ def _logical_func(
 
     def any(
         self,
+        *,
```

Comment:
we can't do this yet, there's still the deprecation warning about this, for now just add overloads as if the deprecation had already taken place

Reply:
Without adding this in the `generic.py`. 
I was getting the error of `Signature of "any" incompatible with supertype "NDFrame"` in mypy.

Removed it now.

Reply:
I think it is okay to ignore this supertype error.